argoverse/utils/make_att_files.py

```diff
@@ -0,0 +1,380 @@
+import sys
+import argoverse
+from argoverse.data_loading.argoverse_tracking_loader import ArgoverseTrackingLoader
+import open3d as o3d
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+import glob
+import scipy
+import scipy.interpolate as interpolate
+import torch.nn.functional as F
+import torch
+import cv2
+import pickle
+
+dict_color = {}
+dict_color["easy"] = (0, 1.0, 0)
+dict_color["far"] = (0, 0.4, 0)
+dict_color["occ"] = (0, 0, 1.0)
+dict_color["fast"] = (0, 1.0, 1.0)
+dict_color["short"] = (0.8, 0.3, 0.3)
+
+list_attritubes = ["short", "occ", "fast", "far", "easy"]
+
+font = cv2.FONT_HERSHEY_SIMPLEX
+fontScale = 1.2
+fontColor = (255, 255, 255)
+lineType = 3
+visualize = True
+check_track_label_folder = True
+
+
+def save_bev_img(path_output_vis, list_bboxes, list_difficulty_att,
+                 dataset_name, log_id, lidar_timestamp, pc,
+                 egovehicle_to_city_se3):
+    """
+    Plot results on bev images and save 
+    """
+    image_size = 2000
+    image_scale = 10
+    img = np.zeros((image_size, image_size, 3))
+    pc = (pc * image_scale)
+    pc[:, 0] += int(image_size / 2)
+    pc[:, 1] += int(image_size / 2)
+    pc = pc.astype('int')
+
+    ind_valid = np.logical_and(
+        np.logical_and(pc[:, 0] >= 0, pc[:, 1] >= 0),
+        np.logical_and(pc[:, 0] < image_size, pc[:, 1] < image_size))
+    img[pc[ind_valid, 0], pc[ind_valid, 1], :] = 0.4
+
+    path_imgs = os.path.join(path_output_vis, 'bev')
+    if not os.path.exists(path_imgs):
+        os.mkdir(path_imgs)
+
+    for bbox, difficulty_att in zip(list_bboxes, list_difficulty_att):
+
+        q0, q1, q2, q3 = bbox['rotation']['w'], bbox['rotation']['x'], bbox[
```

Comment:
can we name these `qw, qx, qy, qz` instead of `q0, q1, q2, q3`, since different libraries order those quaternion elements differently, but w,x,y,z names are always respected?